cmd/utils/selector.go

```diff
@@ -1,514 +0,0 @@
-// Copyright 2022 The Okteto Authors
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package utils
-
-import (
-	"bytes"
-	"context"
-	"errors"
-	"fmt"
-	"io"
-	"os"
-	"runtime"
-	"strings"
-	"text/template"
-
-	"github.com/chzyer/readline"
-	"github.com/juju/ansiterm"
-	"github.com/manifoldco/promptui"
-	"github.com/manifoldco/promptui/list"
-	"github.com/manifoldco/promptui/screenbuf"
-	oktetoLog "github.com/okteto/okteto/pkg/log"
-	"github.com/okteto/okteto/pkg/okteto"
-	"golang.org/x/term"
-)
-
-const (
-	esc        = "\033["
-	showCursor = esc + "?25h"
-)
-
-// ErrInvalidOption is raised when the selector has an invalid option
-var ErrInvalidOption = errors.New("invalid option")
-
-// OktetoSelector represents the selector
-type OktetoSelector struct {
-	Label string
```

Comment:
changed to private all properties of the struct except Label witch is exposed as is used on some logic out of the package, but rest are variables used internally